projects/plugins/jetpack/extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -94,23 +114,7 @@ export default function Edit( { attributes, clientId, context, setAttributes } )
 				</Placeholder>
 			) }
 			<StripeNudge blockName={ BLOCK_NAME } />
-			<InnerBlocks
-				template={ [
-					[
-						'jetpack/button',
-						{
-							element: 'a',
-							passthroughAttributes: {
-								uniqueId: 'uniqueId',
-								url: 'url',
-							},
-						},
-					],
-				] }
-				templateLock="all"
-				__experimentalCaptureToolbars={ true }
-				templateInsertUpdatesSelection={ false }
```

Comment:
Is there a reason why we are "inverting" these two options?

They were added in https://github.com/Automattic/jetpack/pull/18336, so they might as well be obsolete by now, but I'm just wondering.

Looking at the [docs](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/inner-blocks/README.md):
- `__experimentalCaptureToolbars` defaults to false, so if we remove it, it should be turned off, but I still see the plan selector in the inner button. 🤔 
- No idea what `templateInsertUpdatesSelection` really does. 😅

Reply:
Good catch! This wasn't intentional and I think the new props for the inner blocks comes from a Gutenberg copy/paste. However, looks like it's better this way.

> `__experimentalCaptureToolbars` defaults to false, so if we remove it, it should be turned off, but I still see the plan selector in the inner button

Yeah, nothing changes here because we're anyway exposing the block controls to the inner blocks with the `__experimentalShareWithChildBlocks` prop:

https://github.com/Automattic/jetpack/blob/1d8fe8e15a77093936bac79475335a643752433f/projects/plugins/jetpack/extensions/shared/components/product-management-controls/index.js#L60

https://github.com/Automattic/jetpack/blob/1d8fe8e15a77093936bac79475335a643752433f/projects/plugins/jetpack/extensions/shared/components/product-management-controls/toolbar-control.js#L124

> No idea what `templateInsertUpdatesSelection` really does

When enabled, it auto-selects the new inner block, so users can start editing the button text straight away. When disabled, users need to select the inner block to do that.


`templateInsertUpdatesSelection: false`

https://user-images.githubusercontent.com/1233880/183071756-d1ee8395-4833-453d-a84e-5ec9a5bfa4b3.mov


`templateInsertUpdatesSelection: true`

https://user-images.githubusercontent.com/1233880/183071606-051f9dfe-36c9-4992-a3a2-c6157643f913.mov